File: webapp-net/Test/Models/NoCachePageModel.cs
using Sdl.Web.Common.Models;
using Sdl.Web.Common;

namespace Sdl.Web.Modules.Test.Models
{
    [DxaNoCache]
    public class NoCachePageModel : PageModel
    {
        public NoCachePageModel(string id)
                : base(id)
        {
        }
        
    }
}

File: webapp-java/dxa-module-test/src/main/java/com/sdl/dxa/modules/model/TSI2525/NoCachePageModel.java
package com.sdl.dxa.modules.model.TSI2525;

import com.sdl.dxa.caching.NeverCached;
import com.sdl.dxa.caching.NoOutputCache;
import com.sdl.webapp.common.api.model.page.DefaultPageModel;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
@NoOutputCache
@NeverCached
public class NoCachePageModel extends DefaultPageModel {

    public NoCachePageModel() throws ClassNotFoundException {
        TestCacheListenerExtender.registerListeners();
    }
}
